debian/kano-connect.preinst

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# preinst
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
+#
+
+case "$1" in
+    install|upgrade)
```

Comment:
I'd suggest 3 tests here with this new script:

 - On a fresh img build, the fix should work
 - From an older version upgrading to this version, the fix should world
 - Upgrading to a newer version of kano-settings and kano-connect, the fix should still work